cmd/podman/system/connection/add.go

```diff
@@ -236,77 +279,37 @@ func getUDS(cmd *cobra.Command, uri *url.URL) (string, error) {
 			}
 		}
 	}
-
 	var authMethods []ssh.AuthMethod
 	if len(signers) > 0 {
 		var dedup = make(map[string]ssh.Signer)
-		// Dedup signers based on fingerprint, ssh-agent keys override CONTAINER_SSHKEY
 		for _, s := range signers {
 			fp := ssh.FingerprintSHA256(s.PublicKey())
 			if _, found := dedup[fp]; found {
-				logrus.Debugf("Dedup SSH Key %s %s", ssh.FingerprintSHA256(s.PublicKey()), s.PublicKey().Type())
+				logrus.Debugf("Dedup SSH Key %s %s", ssh.FingerprintSHA256(s.PublicKey()), s.PublicKey().Type()) // public key info and validation
 			}
 			dedup[fp] = s
 		}
-
```

Comment:
(This is totally up to Podman maintainers — and maybe you were told to do this, I wouldn’t know — but I generally frown on PRs reformatting code unrelated to the PR itself; the default expectation is for the newly added code to conform to existing style, instead.)

Reply:
I just changed this as a force of habit. I will change it back as there is no need.